daboizzBot/bot.py

```diff
@@ -0,0 +1,94 @@
+
+import discord
+import json
+from discord.ext import commands
+from discord.utils import get
+from discord_slash import SlashCommand
+
+
+def create_bot():
```

Comment:
This is a nicer way to structure the python repo. 

Lets say you want to change or repurpose things this is a lot cleaner.